chess-AI/train.py

```diff
@@ -1,6 +1,8 @@
 """
 Main training program
 """
+from os import path, makedirs
```

Comment:
sorry to pep8 u again but can you just `import os` and then do `os.blah` everywhere you use it?